pkg/controller/kieapp/defaults/defaults_test.go

```diff
@@ -3432,7 +3461,7 @@ func TestMultipleBuildConfigurations(t *testing.T) {
 	env, err := GetEnvironment(cr, test.MockService())
 
 	assert.Nil(t, err, "Error getting prod environment")
-	assert.Len(t, env.Servers, 2, "Expect two KIE Servers to be created based on provided build configs")
+	assert.Len(t, env.Servers, 3, "Expect two KIE Servers to be created based on provided build configs")
```

Comment:
Is this asser correct and should be there 3 server? If so it would be good to correctly update comment as well.

Reply:
it is correct, the build object I've added is a new server object.